libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[configuration-ssl]]
-== Specify SSL settings
+== SSL settings for outputs
```

Comment:
The SSL settings are relevant for other stuff (like kibana), too, so I wouldn't change the title here.